5-infrastructure/business_unit_1/development/variables.tf

```diff
@@ -78,3 +78,21 @@ variable "gke_mci_cluster_cidr_block" {
   description = "The primary IPv4 cidr block for multi-cluster ingress (MCI)."
   default     = "100.64.70.0/28"
 }
+
+variable "enforce_bin_auth_policy" {
+  type        = bool
+  description = "Enable or Disable creation of binary authorization policy"
+  default     = false
+}
+
+variable "bin_auth_attestor_names" {
+  type        = list(string)
+  description = "Binary Authorization Attestor Names set up in shared app_cicd project"
+  default     = ["build", "quality", "security"]
+}
+
+variable "bin_auth_attestor_project_id" {
+  type        = string
+  description = "Project id where binary attestors are created (app_cicd project from shared)"
+  default     = ""
```

Comment:
dev and non-prod have default = "" while prod has none as it is required since bin_auth is recommended only for prod.
Though I can change this if needed